test/prefer-node-remove.js

```diff
@@ -0,0 +1,43 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/prefer-node-remove';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	}
+});
+
+ruleTester.run('prefer-node-remove', rule, {
+	valid: [
+		'foo.parentNode.removeChild(bar)',
+		'this.parentNode.removeChild(foo)',
```

Comment:
I think we should error on these too (line 13 and 14), but just don't auto-fix.

Reply:
Good point. I incorrectly thought that would change the semantics.

Reply:
Oh wow, I didn't even think of it from that point of view. That makes the whole rule a lot simpler 😄.